movidius_ncs_example/src/single_multiple_device_example/multiple_devices_detection.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
+#include <cv_bridge/cv_bridge.h>
+#include <ros/ros.h>
+
+#include <object_msgs/DetectObject.h>
+
+#include <vector>
+#include <dirent.h>
+#include <random>
```

Comment:
please rearrange the header order according to google coding style.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes